src/Services.Remoting/Client/OmexServiceProxyFactory.cs

```diff
@@ -18,9 +25,10 @@ public static class OmexServiceProxyFactory
 					remotingCallbackMessageHandler: handler)));
 
 		/// <summary>
-		/// Binds transport settings to the service proxy factory, for use in secure remoting communication.
+		/// Binds custom transport settings to the service proxy factory.
+		/// If non non-default secure remoting configuration is required, call this before calling OmexServiceProxyFactory.Instance.
 		/// </summary>
-		public static void WithTransportSettings(FabricTransportRemotingSettings transportSettings)
+		public static void WithCustomTransportSettings(FabricTransportRemotingSettings transportSettings)
```

Comment:
I'm open to removing this altogether.
I have only left it here to cover the following case, which is not covered by the default initialisation:
Service A -remoted_call-> Service B -remoted_call-> Service C
Service B cannot use default transport settings for both it's listener and proxy setup (unless it is intentional to use the same cert for client and remote auth).

Reply:
You mean removing the ability to override the defaults entirely? Can you explain further why Service B can't use the defaults in a chained service call?

Reply:
On reading my comment, it was probably a little unclear.
With the Instance, the default behaviour of Service Fabric will be to try and load TransportSettings from the TransportSettings section in the service manifest.
The same is true when creating listeners on the client side.
If a service wants to use the same certs for both it's endpoints (listeners) and connections to upstream services (proxies), the defaults are fine.
If a service needs to use a different way of configuring its transport settings (through code or some other config source) or if they need to configure proxies with a different section to listeners, they can use this WithCustomTransportSettings.

Reply:
> I'm open to removing this altogether. I have only left it here to cover the following case, which is not covered by the default initialisation: Service A -remoted_call-> Service B -remoted_call-> Service C Service B cannot use default transport settings for both it's listener and proxy setup (unless it is intentional to use the same cert for client and remote auth).

I would leave the method then, the scenario you describe in the comment seems plausible and probable enough.